guide/11-performing-network-analyses/guide_to_network_analysis_part1.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>"In order to model transportation networks, Network Dataset<strong>s</strong> are created from source features.." -&gt; make Dataset plural</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/528/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>